include/ccl.h

```diff
@@ -20,21 +20,17 @@ extern "C" {
  *For the latest version of CCL, please visit the github repository: https://github.com/LSSTDESC/CCL
  */
 
-#pragma once
-
+#include "ccl_defs.h"
+#include "ccl_utils.h"
+#include "ccl_config.h"
 #include "ccl_core.h"
+#include "ccl_error.h"
 #include "ccl_constants.h"
 #include "ccl_power.h"
 #include "ccl_cls.h"
-#include "ccl_utils.h"
 #include "ccl_background.h"
-#include "ccl_config.h"
 #include "ccl_correlation.h"
 #include "ccl_massfunc.h"
 #include "ccl_neutrinos.h"
-  
-#include "ccl_placeholder.h"
 
-#ifdef __cplusplus
-}
```

Comment:
Does this file need the C++ wrapper macros?

Reply:
It included ccl_defs.h that defined the macro.

Reply:
I mean does the whole file need to have the starting and ending `CCL_*_DECLS` things?

Reply:
ccl.h doesn't declare anything thus doesn't need the pair.

I've added a dummy section in the file to clarify this.